src/components/Nav/Nav.js

```diff
@@ -17,19 +18,32 @@ const Nav = () => {
   const quitLogin = () => {
     setLoginModal(false);
   };
-
   const handleNavDrawer = () => {
     setNavDrawer(prev => !prev);
   };
+
+  const changeNavStyle = () => {
+    if (window.scrollY >= 80) {
+      SetNavBar(false);
+    } else {
+      SetNavBar(true);
+    }
+  };
+  // throttle을 이용해보려 lodash를 설치했습니다.
+
+  useEffect(() => {
+    window.addEventListener('scroll', changeNavStyle);
+  }, [navBar]);
```

Comment:
- 컴포넌트가 언마운트 될 때에도 scroll이벤트가 발생 할 수도 있어서 clean up 함수를 사용해주면 좋습니다.

```suggestion
  useEffect(() => {
    window.addEventListener('scroll', changeNavStyle);
    return () => window.removeEventListener('scroll', changeNavStyle);
  }, []);
```